src/rgb-hsv.js

```diff
@@ -43,18 +43,19 @@ export function hsv2rgb(hsvH, hsvS, hsvV) {
 	const rgbF = hsvH / 60 - rgbI & 1 ? hsvH / 60 - rgbI : 1 - hsvH / 60 - rgbI;
 	const rgbM = hsvV * (100 - hsvS) / 100;
 	const rgbN = hsvV * (100 - hsvS * rgbF) / 100;
+	const rgbT = hsvV * (100 - (100 - rgbF) * hsvS / 100) / 100;
```

Comment:
Shall be `hsvV * (1 - (1 - rgbF) * (hsvS / 100))`? 
`rgbF` shall not be divided by 100?